appservice/src/createAppService/SiteCreateStep.ts

```diff
@@ -76,23 +92,22 @@ export class SiteCreateStep extends AzureWizardExecuteStep<IAppServiceWizardCont
                     name: 'AzureWebJobsStorage',
                     value: storageConnectionString
                 },
-                {
-                    name: 'FUNCTIONS_EXTENSION_VERSION',
-                    value: 'latest'
-                },
                 {
                     name: 'WEBSITE_CONTENTAZUREFILECONNECTIONSTRING',
                     value: storageConnectionString
                 },
                 {
                     name: 'WEBSITE_CONTENTSHARE',
                     value: fileShareName
-                },
-                {
-                    name: 'WEBSITE_NODE_DEFAULT_VERSION',
-                    value: '6.5.0'
                 }
             ];
+
+            for (const key of Object.keys(this._functionAppSettings)) {
```

Comment:
Object.assign?

Reply:
Was not aware of `Object.assign` - that's nifty. However I don't think I can use it in this case since I'm changing from `{ exampleKey: exampleValue }` to `{ name: exampleKey, value: exampleValue }`